src/app/combobox/index.ts

```diff
@@ -0,0 +1 @@
+export { ComboboxModule } from './combobox.module';
```

Comment:
We typically export * (i.e., everything exported by the module). For example, we should export the component here. 

Even though its available in the module as well, it's sometimes easier to import just the component in order to access its methods.